src/feature.py

```diff
@@ -0,0 +1,169 @@
+from chem_lex.ChemlabTokens import unit_prefix, supported_units
+#import
+from element import Element
+def convertTo(el, num, Unit, newUnit):
```

Comment:
I need you to test this with the chemlab test included and make sure that all still passes and that the tests in there dont blow up